api/src/wfl/api/routes.clj

```diff
@@ -90,31 +89,38 @@
                      :securityDefinitions {:googleoauth {:type "oauth2"
                                                          :flow "implicit"
                                                          :authorizationUrl "https://accounts.google.com/o/oauth2/auth"
-                                                         :scopes {:openid  "open id authorization"
-                                                                  :email   "email authorization"
-                                                                  :profile "profile authorization"}}}
+                                                         :scopes {:openid  "Basic OpenID authorization"
+                                                                  :email   "Read access to your email"
+                                                                  :profile "Read access to your profile"}}}
                      :basePath "/"} ;; prefix for all paths
+           :tags ["Information Authenticated"]
            :handler (swagger/create-swagger-handler)}}]])
 
+(defn endpoint-swagger-auth-processor
+  "Use the same security-info across all /api endpoints"
+  [endpoints]
+  (let [security-info {:swagger {:tags ["Authenticated"]
+                                 :security [{:googleoauth []}]}}]
+    (letfn [(needs-security? [endpoint] (str/starts-with? (first endpoint) "/api"))
+            (write-security-info [method-description] (merge-with merge security-info method-description))
+            (modify-method [method] (zipmap (keys method) (map write-security-info (vals method))))]
+      (vec (map #(if (needs-security? %) (apply vector (first %) (map modify-method (rest %))) %)
+                endpoints)))))
```

Comment:
Super interested in feedback on this. Looks at the endpoints data structure above this might help.